tests/org.eclipse.xtend.ide.tests/src/org/eclipse/xtend/ide/tests/highlighting/XtendHighlightingCalculatorTest.java

```diff
@@ -132,11 +140,13 @@ protected XtendClass clazz(String string) throws Exception {
 	}
 	
 	@Test public void testLiteral() {
+		strictMode = true;
 		expectInsignificant(0, 3);
 		expectInsignificant(3, 2);
+		expectTemplateLineBreak(5, 1);
 		expectInsignificant(6, 1);
+		expectTemplateLineBreak(14, 1); //TODO shouldn't this be POSSIBLE_LINE_BREAK
```

Comment:
I'm confused. In the comment above you said 
"TEMPLATE_LINE_BREAKs are LB that are not visible in the output and thus are not rendered at all. "
So the new line I'm talking about in this TODO is going into the output, no?